src/Altinn.AccessManagement.Persistence/Altinn.AccessManagement.Persistence.csproj

```diff
@@ -5,6 +5,11 @@
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Why do Persistence need these dependencies?

Same goes for the Test project. Test will get it through the accessmanagement project reference and don't need it's own dependency